### DISCUSSION TITLE
[Question][Hw6]

### DISCUSSION THREAD

**Discussion by tanapthetimid** [prosyslab-classroom/cs348-information-security#720] [Q&A]
Name: Tanapoom Sermchaiwong

Hello, I would like to ask a few questions about homework 6.

1. Are we allowed to add more seeds? (either modifying the seed txt file or hard coding it into campaign.ml file)
2. Are we allowed to hard code domain specific knowledge?

**Comment by yeonhee-ryou** [comment]
1. Your local seed txt files won't be used to grade your implementation.
2. What do you mean by "domain specific knowledge"? You can try various strategies and test them with gradescope if it works or not.

**Comment by tanapthetimid** [comment]
Thank you very much.

**Comment by m-spitfire** [comment]
@yeonhee-ryou can I assume that the name of json parser executable is `json_parser`?

**Comment by yeonhee-ryou** [comment]
> @yeonhee-ryou can I assume that the name of json parser executable is `json_parser`?

no
